### DISCUSSION TITLE
Advice on finding missing datasets in Dashboard

### DISCUSSION THREAD

**Discussion by imsuchanooob** [Security-Onion-Solutions/securityonion#11333] [2.4]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

16

### Storage for /

200

### Storage for /nsm

500

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hello, I just recently installed Security Onion's 2.4.10 iso. Currently running the distributed installation with one manager node, one search node, and one forward node. Initially when I ran my installation I had issues connecting to the internet because of our corporate proxy, but I was able to get around that. When I'm looking at my security onion console, I don't have any alerts and my dashboard shows that I'm missing system.syslog, elastic_agent.filebeat, elastic_agent.osquerybeat, and elasticsearch_server.  I validated in Elastic Fleet that the agents are deployed and the policies are enforced. Any help would be appreciated. 

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/145164279/3cd0de06-0bd7-4efb-addf-eaf6a1ed6641)

<img width="1230" alt="image" src="https://github.com/Security-Onion-Solutions/securityonion/assets/145164279/a99ec6cd-9b01-4bd6-85df-3fe7b7c5b85c">

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
> When I'm looking at my security onion console, I don't have any alerts

Have you followed the Troubleshooting Alerts section of the documentation?
https://docs.securityonion.net/en/2.4/suricata.html#troubleshooting-alerts

> my dashboard shows that I'm missing system.syslog, elastic_agent.filebeat, elastic_agent.osquerybeat, and elasticsearch_server

Looking at your first screenshot, it's not saying that it's missing those logs. Instead, it's saying that it does have those logs but that those logs are missing the `event.category` field (which is not necessarily a problem).

**Reply by imsuchanooob** [reply]
Hi Doug, I've tried all the troubleshooting steps in the Alerts section; however, still not seeing any of the alerts. Looking at the discussion board, it seems a lot of people are having the same problem.

**Reply by dougburks** [reply]
> I've tried all the troubleshooting steps in the Alerts section; however, still not seeing any of the alerts.

Are you able to provide any more information about the steps that you took so that we can further help you?

> Initially when I ran my installation I had issues connecting to the internet because of our corporate proxy, but I was able to get around that. 

Is your system able to download NIDS rules through your corporate proxy?

From the Troubleshooting Alerts section of the documentation:
_Check to see if you have a full NIDS ruleset with rules that should specifically alert on the traffic and that those rules are enabled._

What is the output of the following?
```
sudo ls -alh /opt/so/rules/nids/
```
